backend/kernelCI_app/helpers/tests.py

```diff
@@ -0,0 +1,27 @@
+import json
+from typing import Union, TypedDict, Optional
+from kernelCI_app.helpers.filters import UNKNOWN_STRING
+
+
+class EnvironmentMisc(TypedDict):
+    platform: str
+
+
+def handle_environment_misc(misc: Union[str, dict, None]) -> Optional[EnvironmentMisc]:
+    parsed_misc: EnvironmentMisc = {
+        "platform": UNKNOWN_STRING,
+    }
+
+    if misc is None:
+        return parsed_misc
+    elif isinstance(misc, str):
+        try:
+            misc = json.loads(misc)
+        except json.JSONDecodeError:
+            return parsed_misc
```

Comment:
maybe it is a good idea to add the `log_message` here, for now it just prints but in the future will be doing something more

Reply:
And keep returning the default value?

Reply:
omg that was lack of attention, make it the return value back to optional and make it return None